CMakeLists.txt

```diff
@@ -36,18 +36,17 @@ ENDIF()
 
 SET(TF_LIB tensorflow-microlite)
 
+FILE(GLOB COMPILER_HEADERS ${PROJECT_SOURCE_DIR}/src/*.h)
```

Comment:
globbing source files in cmake should generally be avoided: https://cmake.org/cmake/help/v3.15/command/file.html?highlight=glob#file

Reply:
you're making me be less lazy